app/helpers/application_helper.rb

```diff
@@ -88,14 +88,18 @@ def pretty_json(raw_json_string)
   end
 
   def top_menu_items(format_values=nil)
-    tags = Rails.configuration.x.top_menu_items.collect do |menu_item|
-      (url = menu_item['url'] % format_values) if format_values
+    monitoring = { "key" => "monitoring", "url" => "/dashboards/:monitoring",}
```

Comment:
I don't get this initialization. Why don't you create a dictionary, without having individual entries and later push to an array.
You can do this in one single step

Reply:
completely right ! I forgot how to ruby